### PR TITLE
chore: move Role permissions to ClusterRole

### PR DESCRIPTION
**Pull request by MatousJobanek**

### CODE REVIEW

File: deploy/role.yaml [hunk 1]
```diff
@@ -1,81 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
```
Comment (alexeykazakov) [inline deploy/role.yaml]: I guess we can also remove `/deploy/role_binding.yaml` too now?

Comment (MatousJobanek) [reply deploy/role.yaml]: sure 6677503

**Pull request merged**